### ISSUE TITLE
No placeholder for Learner Profile

### ISSUE THREAD

**Issue by ocaisa** [swcarpentry/styles#517]
While preparing for a workshop on lesson design this week, I was surprised to realise that there is no placeholder in the lesson for Learner Profiles. For SWC, for example, you can find this on the website and it is a catchall for the full workshop, but there are many people who are using the lesson style to write their own lessons and don't have such a website. Since the Learner Profile is a key part of backwards lesson design, I think it really deserves a home inside the lesson (perhaps in the `_config.yml` and then rendered in `guide.md`?)

**Comment by maxim-belkin** [comment]
I agree: learner profile(s) are useful (they help lesson maintainers/developers keep the focus of the their target audience; they also help learners understand whether a particular lesson is well suited for them) and reminding lesson developers about them is a good idea. We could have a dedicated file for them (`leaner_profile.md`) or embed them into an existing file (e.g., `guide.md` as you suggested).

**Comment by brownsarahm** [comment]
I agree learner profiles are very useful for lesson development & learner self-assessment/choosing to participate. 

I think a dedicated file for them that is rendered to its own page under extras is possibly better than putting it in guide. 

I did it that way for my [incubator lesson](https://carpentries-incubator.github.io/python-packaging-publishing/learner-profiles/index.html) and it's been handy to have a direct page when preparing information for potential attendees to point them to that page.  If it's on `guide.md` it's an odd place to direct learners trying to decide what to do, since most of the instructor notes would assume more familiarity and have a different audience than the profiles.
